internal/vulkan/vulkan-loader/CMakeLists.txt

```diff
@@ -26,7 +26,12 @@ if(NOT vulkan-loader_POPULATED)
   set(BUILD_SHARED_LIBS_OLD ${BUILD_SHARED_LIBS})
   # Make subproject to use 'BUILD_SHARED_LIBS=ON' setting.
   set(BUILD_SHARED_LIBS ON CACHE INTERNAL "Build SHARED libraries")
+  set(BUILD_WSI_WAYLAND_SUPPORT OFF CACHE INTERNAL "")
```

Comment:
Will this break Wayland?

Reply:
Wayland's not broken but disabled...and I don't know why this is here :D

Reply:
I uhhh

Don't really follow entirely

This sounds bad/wrong but I'm unsure

Reply:
Ah! So this broke stuff because then this is on (which it is by default), Wayland is required otherwise the build fails. Which we can add our own option to toggle this, however without it off by default building with WSL2/WSLg failed.

Reply:
OK we should definitely add a bug that Wayland's gonna commit not alive at the moment with a setup for this

We can add a switch in another PR

Reply:
Created #478 for this